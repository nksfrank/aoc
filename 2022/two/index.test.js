import fs from "node:fs";
import path from "node:path";
import { expect, test } from "vitest";
import { partOne, partTwo } from ".";

const data = `A Y
B X
C Z`.split("\n");

test("part one works", () => {
  expect(partOne(data)).toEqual(15);
});
test("part two works", () => {
  expect(partTwo(data)).toEqual(12);
});
test("live", () => {
  const inputs = fs
    .readFileSync(path.resolve(__dirname, "./input.txt"), "utf-8")
    .split("\n");
  expect(partOne(inputs)).toMatchSnapshot();
  expect(partTwo(inputs)).toMatchSnapshot();
});
