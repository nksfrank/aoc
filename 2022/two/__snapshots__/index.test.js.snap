// Vitest Snapshot v1

exports[`live 1`] = `15422`;

exports[`live 2`] = `15442`;
